laboratory/tests/shared/validators/ModalValidator.ts

```diff
@@ -0,0 +1,43 @@
+import { expect } from '@playwright/test'
+import type { Page } from '@playwright/test'
+import { LOCAL_LAB_URL } from '../constants'
+
+export class ModalValidator {
+  private readonly baseURL = LOCAL_LAB_URL
+
+  constructor(public readonly page: Page) {}
+
+  async isConnected() {
```

Comment:
IMO "expect" represents the behavior of these functions better than "is" because they call "expect". I almost though this function returned a boolean.

```suggestion
  async expectConnected() {
```